src/components/bookList.js

```diff
@@ -0,0 +1,2 @@
+const bookList = () => <div>List of the book</div>;
+export default bookList;
```

Comment:
- [ ] Great effort working on the Bookstore, Initialize project with component milestone, but according to the requirement there are some thing you are currently missing. The requirement state that

1. A `<Books>` component, which contains the following components:
   - `<BookList>` which Renders a list of books using the `<Book>` component

2. `<Book>` which Takes the book data as a prop and renders the title and the author
   - Has a button for removing books: the button wouldn't remove the books yet, this is just a placeholder for now

So therefore kindly create some list of book using the <Book> component and also let it have a remove button but not necessary it is functioning at the moment.